06-Bitfinexed_Spoofy - Prijevod na HR

maaxaam opened the pull request:
Prijevod https://hackernoon.com/meet-spoofy-how-a-single-entity-dominates-the-price-of-bitcoin-39c711d28eb4

maaxaam committed:
Final

Final beta version before publishing of EN version of glossary for the site

maaxaam committed:
Rename Final to Final.md

apetrovic6 committed:
Dodajem članak 00-Tokeni+tut

apetrovic6 committed:
Promjenjen link

apetrovic6 committed:
Rephrased.

apetrovic6 committed:
Minor changes

maaxaam committed:
Create Final.md

Opinion piece about China and Crypto with deeper analysis.

maaxaam committed:
Create Final.md

Hrvatska verzija clanka koji se odnosi na utjecaj Kine na trenutacno stanje trzista.

maaxaam committed:
Update Final.md

maaxaam committed:
Update Final.md

maaxaam committed:
Update Final.md

Swader committed:
Added australia post

Swader committed:
Merge pull request #11 from Swader/post-australia

Added australia post

Swader committed:
Merge pull request #10 from bitfalls/maaxaam-patch-3

Maaxaam patch 3

Swader committed:
Added monero post

Swader committed:
Merge pull request #13 from Swader/post-monerosecret

Added monero post

Swader committed:
Merge pull request #1 from bitfalls/master

Sync

apetrovic6 committed:
Rewrite

apetrovic6 committed:
rewrite

apetrovic6 committed:
screenshot edit

maaxaam committed:
Update Final.md

maaxaam committed:
Delete Final.md

maaxaam committed:
Delete Final.md

maaxaam committed:
Create Final.md

apetrovic6 committed:
Draft

apetrovic6 committed:
-

apetrovic6 committed:
Merge pull request #1 from bitfalls/master

update

Swader committed:
Merge branch 'patch-1' of git://github.com/maaxaam/bitfalls_content into maaxaam-patch-1

msuver12 committed:
Create MarkoSuver

msuver12 committed:
Delete MarkoSuver

msuver12 committed:
Create MarkoSuver

msuver12 committed:
Delete MarkoSuver

msuver12 committed:
01-Ubiq

msuver12 committed:
Delete MarkoSuver

msuver12 committed:
Create final.md

msuver12 committed:
Create final.md

msuver12 committed:
Delete final.md

msuver12 committed:
Create en_EN

msuver12 committed:
Delete en_EN

msuver12 committed:
Delete final.md

msuver12 committed:
Create 01-Ubiq

msuver12 committed:
Delete 01-Ubiq

msuver12 committed:
Create hr_HR

msuver12 committed:
Delete hr_HR

msuver12 committed:
Create 01-Ubiq

msuver12 committed:
Create final.md

msuver12 committed:
Delete 01-Ubiq

msuver12 committed:
Create final.md

msuver12 committed:
Create thumbnail.jpg

msuver12 committed:
Delete thumbnail.jpg

msuver12 committed:
Create 01

msuver12 committed:
Add files via upload

msuver12 committed:
Delete 01

msuver12 committed:
Create 01

msuver12 committed:
Add files via upload

msuver12 committed:
Delete 01

msuver12 committed:
Create final.md

msuver12 committed:
Create final.md

msuver12 committed:
Delete 01.png

msuver12 committed:
Delete 02.PNG

msuver12 committed:
Delete thumbnail.jpg

msuver12 committed:
Delete final.md

msuver12 committed:
Delete final.md

msuver12 committed:
Delete 01.png

msuver12 committed:
Delete 02.PNG

msuver12 committed:
Delete thumbnail.jpg

msuver12 committed:
Create 01

msuver12 committed:
Add files via upload

msuver12 committed:
Delete 01

msuver12 committed:
Create 01

msuver12 committed:
Add files via upload

msuver12 committed:
Delete 01

Swader committed:
Corrections + page

Swader committed:
Merge pull request #16 from bitfalls/maaxaam-patch-1

Maaxaam patch 1

Swader committed:
Giveaway

Swader committed:
Giveaway

Swader committed:
Merge pull request #17 from Swader/master

Merge

maaxaam committed:
Create Final.md

Swader committed:
Merge branch 'master' of git://github.com/maaxaam/bitfalls_content into maaxaam-master

Swader committed:
Corrections

Swader committed:
Merge pull request #19 from Swader/maaxaam-master

Maaxaam master

tyaakow committed:
Merge pull request #1 from Swader/master

syncing

Swader committed:
Update final.md

apetrovic6 committed:
English translation

Swader committed:
Update final.md

Swader committed:
Replaced header image

Swader committed:
Merge pull request #14 from wubwub/01-Ethereum-Roadmap

01 ethereum roadmap

Swader committed:
Merge pull request #2 from bitfalls/master

Updating fork

Swader committed:
Ispravke

Swader committed:
Merge pull request #20 from Swader/master

Ispravke

apetrovic6 committed:
Merge pull request #2 from bitfalls/master

update

apetrovic6 committed:
Draft

apetrovic6 committed:
header

apetrovic6 committed:
Grammar check

apetrovic6 committed:
-

tyaakow committed:
Simon&Speck draft done

Swader committed:
Added blockexplorer post

Swader committed:
Merge pull request #23 from Swader/post-blockexplorer

Added blockexplorer post

Swader committed:
Merge pull request #6 from bitfalls/master

Update

tyaakow committed:
some text written

tyaakow committed:
deleted gif

Swader committed:
Added blockconf post

Swader committed:
Merge pull request #25 from Swader/post-block

Post block

Swader committed:
Merge pull request #7 from bitfalls/master

Update

apetrovic6 committed:
Draft

Swader committed:
Updated giveaway post with winners

Swader committed:
Merge pull request #27 from Swader/master

Update to giveaway

Swader committed:
Update final.md

Swader committed:
Added English ghost post

Swader committed:
Merge pull request #22 from wubwub/03-Crypto-POS-prijevod

03 crypto pos prijevod

Swader committed:
Update final.md

Swader committed:
Update Final.md

tyaakow committed:
wrote some content

tyaakow committed:
prijevod

tyaakow committed:
minor correction

Swader committed:
Post about mining

Swader committed:
Merge pull request #28 from Swader/master

Post about mining

Swader committed:
Merge pull request #8 from bitfalls/master

Updating

Swader committed:
Fixes

Swader committed:
Merge pull request #24 from tyaakow/00-simon-speck

00 simon speck

Swader committed:
Merge pull request #29 from Swader/master

Merge pull request #8 from bitfalls/master

Swader committed:
Fixed links

Swader committed:
Merge pull request #30 from Swader/master

Fixed links

Swader committed:
Merge pull request #9 from bitfalls/master

Sync

maaxaam committed:
Create Final.md

Translation of:
https://medium.com/@bitfinexed/are-fraudulent-tethers-being-used-for-margin-lending-on-bitfinex-5de9dd80f330

maaxaam committed:
05-Tethers_Curious_Story-Prijevod na HR

Prijevod clanka - https://hackernoon.com/the-curious-tale-of-tethers-6b0031eead87 

@Swader

maaxaam committed:
Delete Final.md

maaxaam committed:
05-Tethers_Curious_Story - Prijevod na HR

Prijevod clanka https://hackernoon.com/the-curious-tale-of-tethers-6b0031eead87

maaxaam committed:
Merge pull request #2 from maaxaam/maaxaam-patch-2

05-Tethers_Curious_Story - Prijevod na HR

maaxaam committed:
Delete Final.md

apetrovic6 committed:
English translation

apetrovic6 committed:
Merge pull request #3 from bitfalls/master

-

tyaakow committed:
wrote ipfs article

tyaakow committed:
small correction

msuver12 committed:
Update final.md

msuver12 committed:
Create final.md

msuver12 committed:
Delete final.md

msuver12 committed:
Create 02-Bitcoin_Gold

msuver12 committed:
Delete 02-Bitcoin_Gold

msuver12 committed:
Create final.md

msuver12 committed:
Create images

msuver12 committed:
Add files via upload

msuver12 committed:
Delete images

Swader committed:
Update final.md

Swader committed:
Update final.md

Swader committed:
Create meta.json

Swader committed:
Update final.md

Swader committed:
Update final.md

Swader committed:
Update README.md

Swader committed:
New pages

Swader committed:
Merge pull request #39 from Swader/master

New pages

Swader committed:
Merge branch '02-HW-wallet-misconception' of git://github.com/wubwub/bitfalls_content into 02-HW-wallet-misconception

Swader committed:
Fixes

Swader committed:
Merge pull request #26 from wubwub/02-HW-wallet-misconception

02 hw wallet misconceptions

maaxaam committed:
Update Final.md

maaxaam committed:
Delete Final.md

Swader committed:
removed image

Swader committed:
Merge branch 'master' of https://github.com/maaxaam/bitfalls_content into maaxaam-master

maaxaam committed:
06-Bitfinexed_Spoofy - Prijevod na HR

Translation of: https://hackernoon.com/meet-spoofy-how-a-single-entity-dominates-the-price-of-bitcoin-39c711d28eb4

Swader committed:
Updates

Swader committed:
Fixed glossary

Swader committed:
Fixes

Swader committed:
Merge pull request #41 from bitfalls/maaxaam-master

Maaxaam master

Swader committed:
Merge branch 'maaxaam-patch-2' of https://github.com/maaxaam/bitfalls_content into maaxaam-curious

Swader committed:
Edits

Swader committed:
Merge pull request #44 from bitfalls/maaxaam-curious

Maaxaam curious

Swader committed:
Update Final.md

Swader committed:
Merge pull request #10 from bitfalls/master

Update

apetrovic6 committed:
Initial draft

apetrovic6 committed:
Spacing

apetrovic6 committed:
-

Swader committed:
Adde powpow post

Swader committed:
Merge pull request #51 from Swader/post-powpos

Adde powpow post

Swader committed:
Merge pull request #52 from Swader/master

Merge pull request #10 from bitfalls/master

Swader committed:
Merge pull request #11 from bitfalls/master

Sync

Swader committed:
Bitcoin gold post

Swader committed:
Merge pull request #53 from Swader/post-btg

Post btg

Swader committed:
Merge pull request #12 from bitfalls/master

Sync

msuver12 committed:
Update final.md

msuver12 committed:
Update final.md

msuver12 committed:
Update final.md

Ispravljene sitnije greške, dodane informacije o novom exchangeu i tokenu na platformi, te usklađeno s hrvatskom verzijom.

apetrovic6 committed:
Minor changes

apetrovic6 committed:
Minor changes

Swader committed:
Updated writing tips

Swader committed:
Merge branch 'master' of https://github.com/MarkoSuver/bitfalls_content into MarkoSuver-master

Swader committed:
Merge pull request #55 from Swader/master

Tips

tyaakow committed:
Update final.md

Swader committed:
Updated posts

Swader committed:
Merge pull request #56 from bitfalls/MarkoSuver-master

Marko suver master

Swader committed:
Update final.md

tyaakow committed:
Update final.md

tyaakow committed:
Update final.md

tyaakow committed:
Update final.md

tyaakow committed:
prijevod ipfs

tyaakow committed:
prijevod ipfs

Swader committed:
Fixes

Swader committed:
Merge pull request #57 from bitfalls/MarkoSuver-master

Fixes

Swader committed:
Merge pull request #13 from bitfalls/master

Sync

Swader committed:
Merge pull request #58 from Swader/master

Merge pull request #13 from bitfalls/master

Swader committed:
Merge branch '01-ipfs-internet' of https://github.com/tyaakow/bitfalls_content into 01-ipfs-internet

Swader committed:
Fixes

Swader committed:
Fixes

Swader committed:
Merge pull request #33 from tyaakow/01-ipfs-internet

01 ipfs internet

Swader committed:
Merge pull request #14 from bitfalls/master

Sync

Swader commented:
@maaxaam ovo bi trebalo popraviti - 3 fajla su u PRu umjesto samo jednog. Najbolje pluckat commit koji se odnosi na prijevod van u posebni PR, ili zatvorit ovaj PR i napravit novi iz novog brancha, inače će se overwriteat postojeći sadržaj ako mergamo ovakav.

maaxaam committed:
Delete Final.md

maaxaam committed:
Delete Final.md

maaxaam committed:
Final.md

Original article link: https://hackernoon.com/meet-spoofy-how-a-single-entity-dominates-the-price-of-bitcoin-39c711d28eb4

maaxaam commented:
maknuo sam ova dva koja su bila extra i dodao sa englesku verziju sa orignalnim clankom. 

tako ok?

Max

not exactly what you said, but similar. :P

@Swader

Swader committed:
Create meta.json

Swader committed:
Merge pull request #59 from Swader/master

Merge pull request #14 from bitfalls/master

Swader committed:
Merge pull request #15 from bitfalls/master

Sync

Swader committed:
Removed en_EN, replaced with en_US

Swader committed:
Merge pull request #60 from Swader/master

Folders

Swader committed:
Update final.md

apetrovic6 committed:
Edit

Swader committed:
Merge pull request #16 from bitfalls/master

Sync

Swader committed:
dkuna and ukraine

zpotter committed:
Update README.md ('EN'->'US')

Swader committed:
Merge branch '05-SegWit2x' of https://github.com/wubwub/bitfalls_content into wubwub-05-SegWit2x

Swader committed:
Update

Swader committed:
Update final.md

Swader committed:
Update final.md

Swader committed:
Fix

Swader committed:
Fixes

Swader committed:
Merge pull request #63 from bitfalls/wubwub-05-SegWit2x

Wubwub 05 seg wit2x

Swader committed:
Merge pull request #19 from bitfalls/master

Sync

Swader committed:
Merge pull request #18 from zpotter/patch-1

Update README.md ('EN'->'US')

Swader committed:
Fixes

Swader committed:
Parity post

Swader committed:
Undelete?

Swader commented:
Malo mi se smiruje situacija s konfama pa daj molim te samo rješi sad images, krećemo dalje u ovo. Znači images ne smiju ostati na njihovom CDNu, ono što republishamo mora biti hostano s naše domene. Images u images folder, numerirano od 01 do XX, i tako includano u postu s putanjom tipa `../images/01.png` pa da mogu uploadat i ubacit u post redom. @maaxaam

maaxaam committed:
Create l.l

maaxaam committed:
Delete l.l

maaxaam committed:
Create l.l

maaxaam committed:
Add files via upload

maaxaam committed:
Delete l.l

maaxaam committed:
Update Final.md

maaxaam commented:
Images are up and done. 

@Swader

Swader committed:
Merge branch 'maaxaam-patch-3' of https://github.com/maaxaam/bitfalls_content into maaxaam-patch-3

Swader committed:
Spoofy corrected
